core/src/main/java/lucee/commons/lang/ClassUtil.java

```diff
@@ -161,6 +164,15 @@ public static Class loadClassByBundle(String className, String name, Version ver
 			else ce = new ClassException(
 					"In the OSGi Bundle with the name [" + name + "] and the version [" + version + "] was no class with name [" + className + "] found." + appendix);
 
+			if (SystemUtil.getSystemPropOrEnvVar("lucee.osgi.fatal", "").equalsIgnoreCase("true")) {
+				String msg = "Lucee OSGI error. As LUCEE_OSGI_FATAL=true, shutting down.";
+				CONSOLE_ERR.println(e);
+				CONSOLE_ERR.println(msg);
+				//logDeploy.log(Log.LEVEL_ERROR, "osgi", msg);
+				//shutdownFelix();
+				System.exit(1);
```

Comment:
having a System.exit in the code for me is a no go, because at runtime this stops the servlet engine, so this should never be done nowhere.